meilisearch/tests/index/delete_index.rs

```diff
@@ -26,7 +26,7 @@ async fn create_and_delete_index() {
 async fn error_delete_unexisting_index() {
     let server = Server::new().await;
     let index = server.index("test");
-    let (_, code) = index.delete().await;
+    let (task, code) = index.delete().await;
```

Comment:
Same as above, except you don’t even need to create the index here.
You can directly use the non-existing shared index from here:
https://github.com/meilisearch/meilisearch/blob/7f20c13f3fc374faa0b812d4e21de541ed95746a/meilisearch/tests/common/mod.rs#L143-L149

Reply:
I don't know if it's possible to just call `shared_does_not_exists_index()`, as the returned shared index does not have the necessary methods. only the owned indices appear do have the tested methods if I understand correctly

Reply:
Oh sorry, I thought I answered but it looks like I didn’t 😰 

You’re right, it’s not the most obvious one. I think you could add a method `delete_index_fail` like I did with the [`delete_documents_by_filter_fail`](https://github.com/meilisearch/meilisearch/blob/e78da3528781ab03c9b51a12b153a19af8e68f5b/meilisearch/tests/common/index.rs#L262).
The general idea is that you can try to delete a `Shared` index, but if it succeeds, then the test must fail because other tests may be using it, and we want to see the test that caused the issue first, not the other one.

Thanks for your other commit it’s perfect!

Reply:
ah, perfect, thanks for the guidance!